src/Cortside.RestApiClient/Authenticators/OpenIDConnect/TokenResponse.cs

```diff
@@ -1,6 +1,6 @@
 using System.Text.Json.Serialization;
 
-namespace Cortside.RestSharpClient.Authenticators.OpenIDConnect {
+namespace Cortside.RestApiClient.Authenticators.OpenIDConnect {
     public class TokenResponse {
         [JsonPropertyName("access_token")]
```

Comment:
I can't remember now if I told you about this before, but might be good to convert everything to System.Text.Json.Serialization in case services would try to use these types but use newtonsoft still

Reply:
Thanks -- i will fix this as everything else is expecting Newtonsoft at this point.

Reply:
fixed